runner/src/scenario/ceramic/new_streams.rs

```diff
@@ -207,20 +237,24 @@ async fn instantiate_small_model(
     user: &mut GooseUser,
     store_in_redis: bool,
     conn: Arc<tokio::sync::Mutex<MultiplexedConnection>>,
+    only_once_per_network: bool,
```

Comment:
`only_once_per_network` confused me at first, I was thinking it was once globally, not one writer... maybe we can call it one_sided_test or one_writer_per_network or something? once makes me think literally once and that doesn't seem to be the intention.